python/doc/examples/data_analysis/distribution_fitting/plot_estimate_gev_fremantle.py

```diff
@@ -82,7 +82,7 @@
 view = otv.View(graph)
 
 # %%
-# **Stationary GEV modeling via the profile log-likelihood function**
+# **Stationary GEV mocoefficientsdeling via the profile log-likelihood function**
```

Comment:
oups

Reply:
Ok! Do you understand the error message : 
Warning, treated as error:
/home/maison/openturns-anne/build_doc_gev/python/src/site-packages/openturns/dist_bundle3.py:docstring of openturns.model_copula.DistributionImplementation.drawLowerExtremalDependenceFunction:7:Libellé dupliqué pour l'équation lowerExtremDpceFunc, autre instance dans user_manual/_generated/openturns.SquaredNormal